otelcol/collector.go

```diff
@@ -179,6 +180,9 @@ func (col *Collector) setupConfigurationComponents(ctx context.Context) error {
 	}
 
 	if err = cfg.Validate(); err != nil {
+		if errors.Is(err, errMissingReceivers) {
+			return fmt.Errorf("no receiver configuration specified in collector configuration file")
+		}
```

Comment:
The `errMissingReceivers` is ok.  What we want to change it detecting, during `Validate` that the configuration is completely empty, not just that it is missing receiver configuration.

Reply:
Alright. Let me check inside the cfg.Validate() and  make the changes in there